src/elm/Session/LoggedIn.elm

```diff
@@ -489,7 +436,7 @@ viewHeader ({ shared } as model) profile_ =
                   else
                     text ""
                 ]
-            , div [ class "relative z-20" ]
+            , div [ class "relative z-50" ]
```

Comment:
lol

Reply:
Yeah, was having some issues where the `Feedback` would show up above the profile nav, I guess this makes sure that doesn't happen